src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -741,16 +752,18 @@ else if (block instanceof BlockEndPortal || block instanceof BlockMobSpawner ||
 
     private void setTileEntity(BlockPos pos)
     {
-        TileEntity tileEntity = job.getSchematic().getTileEntity();//TODO do we need to load TileEntities when building?
-        if(tileEntity != null && world.getTileEntity(pos) != null)
+        //TODO do we need to load TileEntities when building?
+        TileEntity tileEntity = job.getSchematic().getTileEntity();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'tileEntity' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)